api/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:17-alpine
+
+WORKDIR /app
+
+COPY . .
+
+RUN npm install
+
```

Comment:
will start runiing at the time of build
i.e we will get node_modeules!

this will run in /app